### PR TITLE
ADBP-1535-P2A

### PR DESCRIPTION
**Pull request by KashifM101**
Link to Jira:
https://101digital.atlassian.net/browse/ADBP-1535
UI Changes:
added privacy policy and term & comndition

### CODE REVIEW

File: src/components/transfer-confirmation-component/index.tsx [hunk 1]
```diff
@@ -133,8 +141,31 @@ const TransferConfirmationComponent = ({
   const [transferStatus, setTransferStatus] = useState<
     "cancelled" | "failed" | "timeout"
   >("timeout");
+  const [documentLink, setDocumentLink] = useState<string>();
+  const [documentTitle, setDocumentTitle] = useState<string>('');
+  const [isShowDocument, setIsShowDocument] = useState<boolean>(false);
   const [showAlert, setAlert] = useState(false);
+  const onViewPrivacyAndPolicy = () => {
+    const privacyLink = remoteConfig().getString('privacyPolicyTemplate');
```
Comment (hiteshdigit101) [inline src/components/transfer-confirmation-component/index.tsx:149]: Why have you used remoteConfig()?

File: src/components/transfer-confirmation-component/index.tsx [hunk 2]
```diff
@@ -133,8 +141,31 @@ const TransferConfirmationComponent = ({
...
+    console.log('onViewPrivacyAndPolicy => privacyLink', privacyLink);
```
Comment (hiteshdigit101) [inline src/components/transfer-confirmation-component/index.tsx:150]: remove console log

File: src/components/transfer-confirmation-component/index.tsx (hunk 1 above)
Comment (KashifM101) [reply src/components/transfer-confirmation-component/index.tsx:149]: so we don’t have to link it only make it touchable?

Comment (hiteshdigit101) [reply src/components/transfer-confirmation-component/index.tsx:149]: You could have asked to Kayathri or need to check scope of the ticket

Comment (KashifM101) [reply src/components/transfer-confirmation-component/index.tsx]: I asked we don’t have anything for it yet.. so only make it clickable.

Comment (KashifM101) [reply src/components/transfer-confirmation-component/index.tsx]: In the new push

**Review by hiteshdigit101 — APPROVED** [review]

**Pull request merged**